api/src/main/k8s.yml

```diff
@@ -2,7 +2,7 @@ database:
   driverClass: org.postgresql.Driver
   user: morpheus
   password: findneo
-  url: jdbc:postgresql://postgres:5432/sre-lab
+  url: jdbc:postgresql://postgres-postgresql.postgres.svc.cluster.local:5432/postgres
```

Comment:
at some point will want to make this pull from a secret or something instead

Reply:
I think we could add a terraform module that sets up the k8 cluster with the needed stuff (namespaces, configmaps, secrets) and the reference those secrets in the yaml

Reply:
yea good call